apps/src/templates/sectionAssessments/StudentMultipleChoiceOverviewTable.jsx

```diff
@@ -0,0 +1,159 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+export const COLUMNS = {
+  NAME: 0,
+  NUM_MULTIPLE_CHOICE_CORRECT: 1,
+  NUM_MULTIPLE_CHOICE: 2,
+  PERCENT_CORRECT: 3,
+  SUBMISSION_TIMESTAMP: 4,
+};
+
+const studentOverviewDataPropType = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  numMultipleChoiceCorrect: PropTypes.number.isRequired,
+  numMultipleChoice: PropTypes.number.isRequired,
+  percentCorrect: PropTypes.string.isRequired,
+  submissionTimestamp: PropTypes.string,
+  submissionStatus: PropTypes.string,
+});
+
+class StudentAssessmentOverviewTable extends Component {
+  static propTypes= {
+    studentOverviewData: PropTypes.arrayOf(studentOverviewDataPropType),
+  };
+
+  state = {
+    [COLUMNS.NAME]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = (selectedColumn) => {
+    this.setState({
+      sortingColumns: sort.byColumn({
+        sortingColumns: this.state.sortingColumns,
+        // Custom sortingOrder removes 'no-sort' from the cycle
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc'
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  submissionTimestampColumnFormatter = (submissionTimeStamp, {rowData}) => {
+    const submissionStatus = rowData.submissionStatus;
+
+    if (submissionStatus === 'Completed') {
+      return (
+        <div>
+          <div>
+            {submissionTimeStamp}
+          </div>
+          <div>
+            <FontAwesome id="checkmark" icon="check-circle"/>
+          </div>
+        </div>
+      );
+    } else {
+      return submissionStatus;
+    }
+  };
+
+  getColumns = (sortable) => {
+    let dataColumns = [
+      {
+        property: 'name',
+        header: {
+          label: i18n.studentName(),
```

Comment:
this (kind of) relates to my comment about string consistency above. i believe there's a `i18n.name()` string already available, so unless the column is spec'd to say "Student Name" rather than just "Name," you could use that

Reply:
Good observation.  The spec for the column label is "Student Name"